crates/forge_main/src/banner.rs

```diff
@@ -1,10 +1,24 @@
 use std::io;
+use std::path::Path;
 
 use colored::Colorize;
+use dirs::home_dir;
+use dotenv::dotenv;
+use forge_display::title::TitleFormat;
 
 const BANNER: &str = include_str!("banner");
 
 pub fn display() -> io::Result<()> {
+    let dotenv_path = dotenv().ok();
+
+    // Load .env and get the path if successful
+    if let Some(path) = dotenv_path {
+        // Replace home path with ~ if possible
+        let display_path = collapse_home_dir(&path);
+
+        println!("{}", TitleFormat::info(format!("Reading {}", display_path)));
+    }
```

Comment:
On second thoughts, it would be better to show this information under `/info` instead of everytime showing it when forge starts.

Reply:
`banner.rs` also isn't the appropriate place handling this information.

Reply:
@tusharmath : okay, I will open a new PR with information under `/info` command response